### PR TITLE
initial pulse3d worker

### PR DESCRIPTION
**Pull request by jperson**

### CODE REVIEW

File: jobs/pulse3d/pulse3d-worker/src/main.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import boto3
+
+from pulse3D.plate_recording import PlateRecording
+from pulse3D.excel_writer import write_xlsx
+from jobs import get_item, EmptyQueue
+
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+logger = logging.getLogger(__name__)
+
+PULSE3D_UPLOADS_BUCKET = os.getenv(UPLOADS_BUCKET_ENV, "test-sdk-upload")
```
Comment (luciipak) [inline jobs/pulse3d/pulse3d-worker/src/main.py:18]: does the env var key need to be a string?

**Review by luciipak — APPROVED** [review]

File: jobs/pulse3d/pulse3d-worker/src/main.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    logger.info(f"No jobs in queue {e}")
+                    return
+                except Exception as e:
+                    logger.exception("Processing queue item failed")
+                    return
+
+
+if __name__ == "__main__":
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(main())
```
Comment (tannermpeterson) [inline jobs/pulse3d/pulse3d-worker/src/main.py:84]: Can't you just use `asyncio.run(main())` in python 3.7+ ?

File: core/lib/jobs/jobs/jobs.py [hunk 3]
```diff
@@ -7,16 +7,16 @@ class EmptyQueue(Exception):
     pass
 
 def get_item(*, queue):
-    QUERY = "DELETE FROM jobs_queue WHERE id = (SELECT id FROM jobs_queue WHERE queue=$1 ORDER BY priority DESC, created_at ASC FOR UPDATE SKIP LOCKED LIMIT 1) RETURNING id, upload_id, created_at, meta"
+    query = "DELETE FROM jobs_queue WHERE id = (SELECT id FROM jobs_queue WHERE queue=$1 ORDER BY priority DESC, created_at ASC FOR UPDATE SKIP LOCKED LIMIT 1) RETURNING id, upload_id, created_at, meta"
 
     def _outer(fn):
         @wraps(fn)
         async def _inner(*, con):
```
Comment (tannermpeterson) [inline core/lib/jobs/jobs/jobs.py:14]: What do outer and inner do here?

File: jobs/pulse3d/pulse3d-worker/src/main.py (hunk 1 above)
Comment (jperson) [reply jobs/pulse3d/pulse3d-worker/src/main.py:18]: yep, my bad

File: jobs/pulse3d/pulse3d-worker/src/main.py (hunk 2 above)
Comment (jperson) [reply jobs/pulse3d/pulse3d-worker/src/main.py:84]: oh, maybe, I'll give it a try

**Pull request merged**